powertools-e2e-tests/pom.xml

```diff
@@ -184,6 +184,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
+                <version>3.1.1</version>
```

Comment:
For "internal" stuff - not the examples - I don't know if it makes sense to pass this down in a variable ?

Reply:
done :-)